omise/omise-android
# Create payment from internet banking

### Objective

To create the internet banking chooser page for a source from chose internet banking.

### Description of changes

In this PR created the InternetBankingChooserFragment to display the list of allowed banks. And when the user chose the bank, it will send a request to `PaymentCreatorRequester` to create a source.

### QA

To test it in the sample app, it needs to create an internet banking method. And after create source successful it should return a source object.

![Screenshot_1568272910](https://user-images.githubusercontent.com/2638321/64764250-df889d00-d56b-11e9-81ff-b71eff14f3ce.png)

![Screenshot_1568272905](https://user-images.githubusercontent.com/2638321/64764249-df889d00-d56b-11e9-9be9-58401ae0ccad.png)
